daemon/main.go

```diff
@@ -29,12 +32,35 @@ var (
 )
 
 var (
-	client *elastic.Client
-	aut    *auth.Auth
+	client                 *elastic.Client
+	aut                    *auth.Auth
+	accessControl          map[string]UserAccess
```

Comment:
This is obv. only going to work with a single instance running. Good enough for now